lighthouse-core/report/templates/report-template.html

```diff
@@ -187,9 +183,7 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
                   {{/if}}
 
                   <div class="subitem-result">
-                    {{#if subItem.comingSoon}}
-                          <span class="subitem-result__unknown score-unknown-bg">N/A</span>
-                    {{else if subItem.error}}
+                    {{#if subItem.error}}
```

Comment:
We still need the unknown error background image. https://github.com/GoogleChrome/lighthouse/commit/a067d3060b4fb79ebb6ca11e412c2c12699107f3#diff-181e527364f32304fefacb83683da1a7R193.